packages/perseus/src/widgets/interactive-graphs/graphs/linear.test.tsx

```diff
@@ -50,21 +56,36 @@ describe("Linear graph screen reader", () => {
         );
     });
 
-    test("should have aria labels for both points on the line", () => {
+    test("should have aria labels and describedbys for both points and grab handle on the line", () => {
```

Comment:
this might need to be 4 separate tests: point + aria label, point + description, grab handle + aria label, grab handle + description